EnhancedMediaPlayer/MediaPlayerViewController.swift

```diff
@@ -99,3 +124,9 @@ public class MediaPlayerViewController: UIHostingController<MediaPlayerView> {
     }
 }
 
+extension MediaPlayerViewController {
+    enum Constants {
+        static let barUpdateInterval: CGFloat = 1
```

Comment:
and always specify the time when declaring time variables, eg: 

```suggestion
        static let barUpdateIntervalInSec: CGFloat = 1
```

```suggestion
        static let barUpdateIntervalInMs: CGFloat = 1000
```